sdk/src/client.rs

```diff
@@ -169,9 +169,6 @@ pub trait SyncClient {
 
     /// Calculate the fee for a `Message`
     fn get_fee_for_message(&self, message: &Message) -> Result<u64>;
-
-    /// Get a new blockhash after the one specified
-    fn get_new_latest_blockhash(&self, blockhash: &Hash) -> Result<Hash>;
```

Comment:
Was this ever released?  Looks like no, but we should deprecate instead, if it was

Reply:
Never was released